upgrades/3.x/README.adoc

```diff
@@ -58,6 +60,10 @@ You may be required to perform manual actions as part of the upgrade.
 WARNING: Be sure to run scripts on the correct database since `gravitee` is not always the default database!
 Check your db name by running `show dbs;`
 
+include::https://raw.githubusercontent.com/gravitee-io/release/master/upgrades/3.x/3.12.0/README.adoc[leveloffset=+1]
```

Comment:
Not necessary in this PR, will be done with https://github.com/gravitee-io/release/pull/337

Reply:
ok, removed it